lib/media/simple_abr_manager.js

```diff
@@ -159,6 +159,32 @@ shaka.media.SimpleAbrManager.prototype.getInitialVideoTrackId = function() {
 
 
 /**
+ * Select the specified video track.
+ *
+ * @param {shaka.player.VideoTrack} track the track to the switch to
+ *
+ * @protected
+ * @expose
```

Comment:
"expose" can be dropped here since this method won't be called by the application.